Symfony/CS/Fixer/Contrib/AlignEqualsFixer.php

```diff
@@ -24,49 +24,66 @@ class AlignEqualsFixer extends AbstractAlignFixer
     /**
      * {@inheritdoc}
      */
-    public function fix(\SplFileInfo $file, $content)
+    public function fix(\SplFileInfo $file, Tokens $tokens)
     {
-        list($tokens, $deepestLevel) = $this->injectAlignmentPlaceholders($content);
+        $this->deepestLevel = 0;
 
-        return $this->replacePlaceholder($tokens, $deepestLevel);
+        // This fixer works partially on Tokens and partially on string representation of code.
+        // During the process of fixing internal state of single Token may be affected by injecting ALIGNABLE_PLACEHOLDER to its content.
+        // The placeholder will be resolved by `replacePlaceholder` method by removing placeholder or changing it into spaces.
+        // That way of fixing the code causes disturbances in marking Token as changed - if code is perfectly valid then placeholder
+        // still be injected and removed, which will cause the `changed` flag to be set.
+        // To handle that unwanted behavior we work on clone of Tokens collection and then override original collection with fixed collection.
+        $tokensClone = clone $tokens;
+
+        $deepestLevel = $this->injectAlignmentPlaceholders($tokensClone);
+        $content = $this->replacePlaceholder($tokensClone, $deepestLevel);
```

Comment:
Done, thanks!

We have 2 aligning fixers. One has `deepestLevel` as property, one calc it and returns as variable.
Just standardize the code between that 2 fixers.